src/containers/MyNdla/Folders/FolderActions.tsx

```diff
@@ -210,6 +218,24 @@ const FolderActions = ({ selectedFolder, setFocusId, folders, inToolbar = false,
       },
     };
 
+    const deleteLink: MenuItemProps = {
+      icon: <Cross />,
+      text: t("myNdla.folder.sharing.button.unSaveLink"),
+      type: "danger",
+      ref: unLinkRef,
+      onClick: async () => {
+        await unFavoriteSharedFolder({
+          variables: {
+            folderId: selectedFolder.id,
+          },
+        });
+        addSnack({
+          id: "linkRemoved",
+          content: t("myNdla.folder.sharing.removeLink"),
```

Comment:
Her må du bruke unSavedLink.
```suggestion
          content: t("myNdla.folder.sharing.unSavedLink", { name: selectedFolder.name }),
```